src/scripts/macros/raise-a-shield.ts

```diff
@@ -34,7 +34,10 @@ export async function raiseAShield(options: ActionDefaultOptions): Promise<void>
             return false;
         }
 
-        if (shield?.isBroken === false) {
+        if (shield?.isDestroyed) {
+            ui.notifications.warn(localize("ShieldIsDestroyed", { actor: speaker.alias, shield: shield.name }));
```

Comment:
we don't have that localization entry

Reply:
![image](https://github.com/foundryvtt/pf2e/assets/651462/273669eb-df2f-44a1-9db1-e22315624f47)

Reply:
Ah, sorry, I forgot it was moved to a different file.